templates/about.html

```diff
@@ -35,7 +35,7 @@
       <p>At Cloudflare, we believe that open source, community-driven projects like CDNJS are the tools upon which the future of the internet will be built. Cloudflare is proud to provide the global CDN infrastructure that will help power that future.</p>
       <br />
       <a href="https://www.algolia.com/?utm_source=cdnjs&utm_medium=link&utm_campaign=cdnjs_about" target="_blank">
-        <img alt="algolia" style="margin-bottom: 20px;" src="/img/algolia256x80.png" width="192" height="60"/>
+        <img alt="algolia" style="margin-bottom: 20px;" src="/img/algolia.svg" width="192" height="60"/>
```

Comment:
The hight/width attribute, though it won't affect how it display, but it makes no sense now IMO.